### ISSUE TITLE
Missing highlighting and grammar submodule in `3.0.0` sources

### ISSUE THREAD

**Issue by zm-cttae** [Gimly/vscode-matlab#179]
The publish missed the git submodule for the grammar @Gimly

Need to `git submodule update --init` and republish as 3.0.1 

(Hope the meeting with Mathworks went well!)

**Comment by ghost** [comment]
Temporary user fix:
- Shell or PWSH: `cd ~/.vscode/extensions/gimly81.matlab-3.0.0 && git init && git remote add origin https://github.com/Gimly/vscode-matlab.git && git submodule update --init`
- Powershell: `cd ~/.vscode/extensions/gimly81.matlab-3.0.0; git init; git remote add origin https://github.com/Gimly/vscode-matlab.git; git submodule update --init`

**Comment by Gimly** [comment]
Published 3.0.1, should be OK now.

**Comment by Gimly** [comment]
@zm-cttae Yes, the meeting with Mathworks went well, there will be news quickly about the future of the extension.

**Comment by zm-cttae** [comment]
Confirmed it's fixed in 3.0.1

**Comment by zm-cttae** [comment]
We can fix the regression by adding `"postinstall": "git submodule update --init"`.

**Comment by zm-cttae** [comment]
(Gentle nudge to [make release tags](https://github.com/Gimly/vscode-matlab/releases/new) for `e609168` - 3.0.0 - and `38ed7dc` -  3.0.1!)
